CHANGELOG.md

```diff
@@ -1,10 +1,30 @@
 # Fluidcontent Change log
 
+upcoming
+------------------
+
+- :exclamation: Support of [TYPO3 6.2 dropped](https://github.com/FluidTYPO3/fluidcontent/commit/f1df9a4b7ee563335c8f1feeb7925e6f820a7f19)
```

Comment:
Nono, just that this line where we say we drop 6.2 should also say that if you for some reason need to continue using 6.2, there is a legacy branch available. We don't want to maintain a changelog for that branch.